test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -67,15 +70,24 @@ func BenchmarkScheduling(b *testing.B) {
 // PodAntiAffinity rules when the cluster has various quantities of nodes and
 // scheduled pods.
 func BenchmarkSchedulingPodAntiAffinity(b *testing.B) {
+	tests := []struct{ nodes, existingPods, minPods int }{
+		{nodes: 500, existingPods: 100, minPods: 400},
+		{nodes: 5000, existingPods: 1000, minPods: 1000},
```

Comment:
since the pod as anti affinity to each other, the number of pods to schedule can't exceed the number of nodes (the topology used in the test)

Reply:
Ack. Can you add the comment in the code?